utilities/navx.py

```diff
@@ -1,43 +1,41 @@
 import math
 
-import wpilib
 from robotpy_ext.common_drivers.navx.ahrs import AHRS
-from robotpy_ext.common_drivers.navx.registerio_spi import RegisterIO_SPI
+from wpilib.interfaces import PIDSource
 
-from utilities.functions import constrain_angle
 
+class NavX:
+    """Wrapper of robotpy navx to be inline with BNO055 api.
+    """
```

Comment:
D200 One-line docstring should fit on one line with quotes

https://www.python.org/dev/peps/pep-0008/#documentation-strings
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Also, this should be capitalised correctly, and have correct grammar.